src/App.js

```diff
@@ -3,13 +3,107 @@ import Form from "./Form";
 import "./App.css";
 
 class App extends React.Component {
+  constructor(){
+    super();
+    this.state = {
+      userInput: [],
+      operation: "",
+      result: "",
+    }
+  }
+
+  handleClick = (event) => {
```

Comment:
```suggestion
  handleSubmit = (event) => {
```
Since this is attached to a form `onSubmit` event, you should call this event handler `handleSubmit` instead of `handleClick`.